mpas_analysis/test/test_climatology.py

```diff
@@ -353,7 +354,6 @@ def test_cache_climatologies(self):
         #        to update the aggregated climatology file because no
         #        additional years were processed and the file was already
         #        complete for the span of years present
-        test2 = (2, ['years0002-0003', 'year0002'], [True, False])
```

Comment:
This is an old version of `test2` that doesn't get used (test2 is defined above and added to the list of tests).